script/build-rules.js

```diff
@@ -588,7 +589,37 @@ presets(root).then((presetObjects) => {
           for (fileName in fixtures) {
             if (own.call(fixtures, fileName)) {
               const fixture = fixtures[fileName]
-              const label = inspect(JSON.parse(setting))
+              /** @type {{settings: Record<string, unknown>, options: unknown}} */
+              const {settings, options} = JSON.parse(setting)
+              const whenConfiguredWith = settings
+                ? Object.entries(settings)
+                    .flatMap(
+                      ([key, value]) =>
+                        /** @type {Array<PhrasingContent>} */ ([
+                          {
+                            type: 'link',
+                            url: `https://github.com/remarkjs/remark/tree/main/packages/remark-stringify#options${key.toLowerCase()}`,
+                            title: null,
+                            children: [
+                              {
+                                type: 'inlineCode',
+                                value: `settings.${inspect({
+                                  [key]: value
+                                }).slice(2, -2)}`
+                              }
+                            ]
+                          },
+                          {type: 'text', value: ', '}
+                        ])
+                    )
+                    .slice(0, -1)
+                : options !== true &&
+                  /** @type {Array<PhrasingContent>} */ ([
+                    {
+                      type: 'inlineCode',
+                      value: inspect(options)
+                    }
+                  ])
```

Comment:
I think this addition should become a new case! Let the previous behavior persist, and add a new branch, so that it results in another phrase. E.g., something like:

```
When the rule is not configured, but when the processor’s [`settings`]()
include [`xxx: 'yyy'`]().
```

Or so?